xbmc/FileItem.h

```diff
@@ -149,6 +149,7 @@ class CFileItem :
   void ToSortable(SortItem &sortable, const Fields &fields) const;
   bool IsFileItem() const override { return true; };
 
+  bool IsEmpty();
```

Comment:
better this this completely, not need here

Reply:
pardon, could you please rephrase it, I didn't get it.

Reply:
only change code that needs to be changed for this pr. you don't call IsEmpty, hence not need to add this method

Reply:
Oh, sorry
I want to call it on wakeup to skip playFile because it still does alot of thing before it does figure out that there is nothing to play.

Reply:
if player did play someting before sleep, currentFile can't be empty